task1_3.py

```diff
@@ -0,0 +1,30 @@
+"""This function adds an element to a list inside a dict of lists. Rewrite it
+to use a try-except statement which handles a possible KeyError if the list
+with the name provided doesn’t exist in the dictionary yet, instead of
+checking beforehand whether it does. Include else and finally clauses in your
+try-except block.
+"""
+
+
+def add_to_list_in_dict(thedict, listname, element):
+    try:
+        try:
+            lst = thedict[listname]
+        except TypeError:
+            raise TypeError("Input correct arguments.")
```

Comment:
you can print error and just raise without anything

Reply:
solved.